requirements.txt

```diff
@@ -1,2 +1,3 @@
 click
+numpy
```

Comment:
The user of requirements.txt is Travis-CI, so let's make this very specific like `numpy~=1.15`